templates/shibboleth/metadata_providers/_metadata_providers_head.erb

```diff
@@ -63,6 +63,10 @@
                     xmlns="urn:mace:shibboleth:2.0:metadata"
                     metadataURL="http://md.incommon.org/InCommon/InCommon-metadata.xml"
                     backingFile="%{idp.home}/metadata/InCommon-metadata.xml"
+                    <%- if @proxy_host -%>
+                    proxyHost="<%= @proxy_host %>"
+                    proxyPort="<%= @proxy_port %>"
```

Comment:
you should have a seperate if for proxy_port, since you can choose not to set this independently....